rskj-core/src/main/java/co/rsk/config/RskSystemProperties.java

```diff
@@ -254,32 +254,58 @@ public List<ModuleDescription> getRpcModules() {
             return modules;
         }
 
-        List<? extends ConfigObject> list = configFromFiles.getObjectList("rpc.modules");
+        ConfigValue modulesConfig = configFromFiles.getValue("rpc.modules");
```

Comment:
Please extract a constant for `"rpc.modules"`

Reply:
yep